### PR TITLE
feat(badge): add `kv-badge` component

### PR DESCRIPTION
**Pull request by djcouto**
#### Badge 
Screenshot 2022-07-29 at 10 16 40

#### Action Button Icon with Badge
Screenshot 2022-07-29 at 10 17 02

### CODE REVIEW

**Comment by ricardogoncalves89** [comment]
@djcouto Please add some screenshots to the description.

**Comment by ricardogoncalves89** [comment]
@djcouto this badge should have a state? and should be used inside the `kv-tree-item` maybe. Please evaluate this possibility

**Review by miguelpinto98 — APPROVED** [review]
👍

**Review by miguelpinto98 — CHANGES_REQUESTED** [review]
Export the component in angular library

**Review by miguelpinto98 — APPROVED** [review]
🚀

**Review by pedrorebelocosta — APPROVED** [review]

**Review by ricardogoncalves89 — APPROVED** [review]

File: packages/ui-components/src/components/tree-item/readme.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# *<kv-tree-item>*
-
-
+# _<kv-tree-item>_
```
[the remark this answers is not in the document]
Comment (djcouto) [reply packages/ui-components/src/components/tree-item/readme.md:1]: it's a local linter config, I'll reset this.

**Pull request merged**